lib/include/openamp/rpmsg.h

```diff
@@ -325,6 +325,31 @@ int rpmsg_create_ept(struct rpmsg_endpoint *ept, struct rpmsg_device *rdev,
 		     const char *name, uint32_t src, uint32_t dest,
 		     rpmsg_ept_cb cb, rpmsg_ns_unbind_cb ns_unbind_cb);
 
+
+/**
+ * rpmsg_create_anon_ept - like rpmsg_create_ept but without name
+ *
+ * Create a RPMsg endpoint, initialize it with source address,
+ * remoteproc address and endpoint callback, and register it to
+ * the RPMsg device.
+ *
+ * @ept: pointer to rpmsg endpoint
+ * @src: local address of the endpoint
+ * @dest: target address of the endpoint
+ * @cb: endpoint callback
+ *
+ * In essence, an rpmsg endpoint represents a listener on the rpmsg bus, as
+ * it binds an rpmsg address with an rx callback handler.
+ */
+
+static inline int rpmsg_create_anon_ept(struct rpmsg_endpoint *ept,
```

Comment:
Why it needs a function to create endpoint without a name? if the endpoint creation function itself allows empty name.
I am fine that to have endpoint without name assuming it is static endpoint. I think it ok to have static endpoint even when NS feature is enabled, but just I don't need it requires a specific wrapper for it. @arnop2, do you have any concern of it?

Reply:
Just make the creation of static point a little bit easier and visible, and align with the kernel api rpmsg_create_ept.

Reply:
I still doubt if it is necessary to define such a function and it is only used for a subset of static endpoints.

Reply:
Ok, I will remove rpmsg_create_anon_ept, since the nameless endpoint is a seldom used feature.